contracts/test/helpers/ProposalTest.sol

```diff
@@ -103,4 +135,38 @@ contract ProposalTest is CompoundGovernorTest {
         uint256 _proposalId = _submitProposal(_proposer, _proposal);
         _failProposal(_proposalId);
     }
+
+
+ 
+    function _buildNewGovernorSetVotingDelayProposal(uint48 _amount) internal view returns (Proposal memory _proposal) {
+        address[] memory _targets = new address[](1);
+        _targets[0] = address(governor);
+
+        uint256[] memory _values = new uint256[](1);
+        _values[0] = 0;
+
+        bytes[] memory _calldatas = new bytes[](1);
+        _calldatas[0] = _buildProposalData("setVotingDelay(uint48)", abi.encode(_amount));
+
+        _proposal = Proposal(_targets, _values, _calldatas, "Set New Voting Delay on New Compound Governor");
+    }
+
+    function _buildAndSubmitOldGovernorSetVotingDelayProposal(uint256 _proposerIndex, uint _amount) internal returns (uint256 _proposalId) {
```

Comment:
does this belong here? would the consumer of this method be better off using `BravoUpgradeTest`?  feel free to push back